components/CreateNewUserAccount/CreateButton.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { Button, Dialog, DialogActions, DialogContent, DialogContentText, DialogTitle } from '@mui/material';
+
+const CreateButton = () => {
+    const [open, setOpen] = React.useState(false);
+
+    const handleClickOpen = () => {
+      setOpen(true);
+    };
+  
+    const handleClose = () => {
+      setOpen(false);
+    };
+  
+    return (
+        <div>
+        <Button variant="contained" onClick={handleClickOpen}>
+          CREATE
+        </Button>
```

Comment:
We should disable this button if any of the fields are blank since all fields should be required